index.html

```diff
@@ -4817,6 +4789,71 @@ <h2>Content Integrity Protection</h2>
 hash for the context as registered in the DID specification registries
 (see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
       </p>
+    <section>
+      <h2>Persistence</h2>
+      <p>
+    </section>
+DIDs are designed to be persistent such that a controller need not rely
+upon a single trusted third party or administrator to maintain their
+identifiers. No administrator can take control away from the controller,
+nor can an administrator prevent their identifiers' use for any particular
+purpose such as authentication, authorization, and attestation. No
+third party can act on behalf of a controller to remove or render
+inoperable an individual's (or an organization's) identifier without the
+controller's consent.
+      </p>
+      <p>
+However, it is important to note that in all DID Methods that enable 
+cryptographic proof-of-control, the means of proving control can always be 
+transferred to another party by transferring the cryptographic secrets.
+Therefore, it is vital that systems relying on the persistence of an 
+identifier over time regularly check to ensure that the identifier is, in
+fact, still under the control of the intended party.
+      </p>
+      <p>
+Unfortunately, it is impossible to determine from the cryptography alone
+whether or not the private key material associated with a given proof 
+mechanism has been compromised. It may well be that the expected controller still has access to the private keys—and as such can execute a
```

Comment:
```suggestion
mechanism has been compromised. It may well be that the expected controller
still has access to the private keys—and as such can execute a
```